scripts/README.md

```diff
@@ -0,0 +1,25 @@
+## ✍️ Scripts
+
+This folder contains convenience functions to run certain tasks. It also contains
+scripts to runs pipelines which are outside this repository, some of them in 
+private Wellcome repositories.
+
+### Evaluation dag & getting grants
+The main dvc DAG has an evaluation independent dag for one of the models (MeSH). To run this,
+you need to be on the Wellcome VPN, you need to install private requirements 
```

Comment:
do you mean part of WellcomeML org in Github?

Reply:
Nope, I mean Wellcome VPN, to query 42. You will also need to be part of the org